### PR TITLE
Use the actual Issuer and Subject DER encodings for all comparison operations

### LINKED ISSUE #184: Use actual DER encodings of Issuer and Subject fields for comparisons/differences
In https://bugzilla.mozilla.org/show_bug.cgi?id=1675813#c4, the Issuer/Subject entry `CN=GlobalSign,O=GlobalSign,OU=GlobalSign ECC Root CA - R5,Ae5fInnr9AhpWVIjkw==` was being repeated into OneCRL, because OneCRL already contains an entry `IS: CN=GlobalSign,OU=GlobalSign ECC Root CA - R5,O=GlobalSign,Ae5fInnr9AhpWVIjkw==`.

Note that the order already in OneCRL is `CN, OU, O` whereas the order coming from the CCADB is `CN, O, OU`.

It's unclear as-of-yet where the reordering is happening, but we need to ensure the order is consistent for comparisons, as they are logically the same.

### PR DESCRIPTION
**Pull request by jcjones**
Fixes #184

### CODE REVIEW

**Review by mozkeeler — APPROVED** [review]
Looks good - just a couple of minor comments.

File: ccadb2OneCRL/utils/utils_test.go [hunk 1]
```diff
@@ -4,88 +4,4 @@
...
-func TestCmpOID(t *testing.T) {
-	for _, data := range cmpData {
-		got := CmpOID(data.left, data.right)
-		if got != data.want {
-			t.Errorf("unexpected comparison answer, wanted %v got %v.", data.want, got)
-			t.Errorf("left: %v, right %v", data.left, data.right)
-		}
-	}
-}
+import ()
```
[the remark this answers is not in the document]
Comment (jcjones) [reply ccadb2OneCRL/utils/utils_test.go:7]: Mmm, adding Base64 decoder tests to make sure that works right. :)

File: ccadb2OneCRL/onecrl/onecrl.go [hunk 2]
```diff
@@ -253,15 +249,10 @@ func FromCCADB(c *ccadb.Certificate) (*Record, error) {
 	return record, nil
 }
 
-func parseRDNS(rdns string) (*pkix.RDNSequence, error) {
+func parseRDNS(rdns string) ([]byte, error) {
```
[the remark this answers is not in the document]
Comment (jcjones) [reply ccadb2OneCRL/onecrl/onecrl.go:252]: Good idea.

**Review by mozkeeler — APPROVED** [review]
lgtm

**Pull request merged**